packages/devtools/lib/src/memory/memory.dart

```diff
@@ -159,17 +198,21 @@ class MemoryScreen extends Screen with SetStateMixin {
               div()..flex(),
               div(c: 'btn-group collapsible-700 flex-no-wrap margin-left')
                 ..add(<CoreElement>[
+                  vmSearchField,
+                  vmMemorySearchButton,
                   vmMemorySnapshotButton,
                   resetAccumulatorsButton,
                   filterLibrariesButton,
                   gcNowButton,
                 ]),
             ]),
         ]),
+      heapPopup, // Auto-complete popup
```

Comment:
nit: maybe call it heapAutocompletePopup so no comment is needed?

Reply:
Done